api/group.go

```diff
@@ -6,10 +6,10 @@ import (
 )
 
 type Group struct {
-	ID         uid.ID `json:"id"`
+	ID         uid.ID `json:"id" note:"Group ID" example:"gauEdoYCEU"`
 	Name       string `json:"name" note:"Name of the group" example:"admins"`
-	Created    Time   `json:"created"`
-	Updated    Time   `json:"updated"`
+	Created    Time   `json:"created" note:"Date the group was created"`
```

Comment:
Should this have an example time to match the destination struct?

Reply:
No, the times generate a valid example on their own. and it  seems to be good.